uberenv.py

```diff
@@ -342,6 +358,18 @@ def __init__(self, opts, extra_opts):
 
         print("[spack spec: {}]".format(self.opts["spec"]))
 
+
+    def append_path_to_packages_paths(self, path, errorOnNonexistant=True):
+        path = pabs(path)
+        if not os.path.exists(path):
+            if errorOnNonexistant:
+                print("[ERROR: Given path in 'spack_packages_path' does not exist: {0}]".format(path))
+                sys.exit(1)
+            else:
+                return
+        self.packages_paths.append(pabs(path))
```

Comment:
path is already pabs(path).
```suggestion
        self.packages_paths.append(path)
```